Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
I noticed that also the forward part of the tracker changed in the meantime, I guess this should be:

```xml
     <!-- single layer close to ExCal -->
     <constant name="FwdTracker_rmin1" value="40*mm"/>
     <constant name="FwdTracker_rmax1" value="1173*mm"/>
     <constant name="FwdTracker_z1" value="7500*mm"/>
     <!-- the rest of the tracking layers inside the forward solenoid / shielding -->
     <constant name="FwdTracker_rmin2" value="60*mm"/>
     <constant name="FwdTracker_rmax2" value="1556*mm"/>
      <constant name="FwdTracker_z2" value="3000*mm"/>
      <constant name="FwdTracker_zOffset" value="13000*mm"/>
```

Can you please double-check @vvolkl / @drasal?